Broken Imports

I think I broke the library as now there are errors due to the utils directory added in the last PR. I think this is because I forgot to add a `__init__` file for the utils folder. I'll create a PR to fix this as locally this seems to fix the issue.

Demo for what's broken is [here](https://colab.research.google.com/gist/aflah02/7f06a9bf29dd0edc5d2aeff9e55c4633/untitled16.ipynb)

@mattdangerw I'm not entirely sure how to fix this as if I add a `__init__` file then tests fail citing 
```
keras_nlp/utils/tensor_utils_test.py:16: in <module>
    from tensor_utils import tensor_to_string_list
E   ModuleNotFoundError: No module named 'tensor_utils'
```

--- a/keras_nlp/utils/__init__.py
+++ b/keras_nlp/utils/__init__.py
@@ -11,5 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from keras_nlp.utils.tensor_utils import tensor_to_string_list

--- a/keras_nlp/utils/__init__.py
+++ b/keras_nlp/utils/__init__.py
@@ -1,0 +1,15 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_nlp.utils.tensor_utils import tensor_to_string_list

--- a/keras_nlp/utils/tensor_utils_test.py
+++ b/keras_nlp/utils/tensor_utils_test.py
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 import tensorflow as tf
-from tensor_utils import tensor_to_string_list
+
+from keras_nlp.utils.tensor_utils import tensor_to_string_list
 
 
 class TensorToStringListTest(tf.test.TestCase):

